OmsAgent/omsagent.py

```diff
@@ -30,12 +31,13 @@
 ExtensionShortName = "OmsAgentForLinux"
 
 PackagesDirectory = "packages"
-BundleFileName = 'omsagent-1.2.0-75.universal.x64.sh'
+BundleFileName = 'omsagent-1.3.0-12.universal.x64.sh'
```

Comment:
now that Narine will probably publish 1.3.15.1 before you, you should use that bundle (and change the extension version to 1.3.15.2)